mne/event.py

```diff
@@ -1422,3 +1424,82 @@ def get_condition(self, raw, condition=None, stim_channel=None, mask=None,
             conds_data.append(dict(events=cat_t0, event_id=cat_id,
                                    tmin=tmin, tmax=tmax))
         return conds_data[0] if len(conds_data) == 1 else conds_data
+
+
+def match_event_names(event_names, keys, *, on_missing='raise'):
+    """Search a collection of event names for matching (sub-)groups of events.
+
+    This function is particularly helpful when using grouped event names
+    (i.e., event names containing forward slashes ``/``). Assuming for example
```

Comment:
Just add "see example below" and move the example to Example section. Then let's merge.